platforms/common/include/px4_platform_common/px4_work_queue/WorkQueueManager.hpp

```diff
@@ -48,7 +48,7 @@ struct wq_config_t {
 
 namespace wq_configurations
 {
-static constexpr wq_config_t rate_ctrl{"wq:rate_ctrl", 1600, 0}; // PX4 inner loop highest priority
+static constexpr wq_config_t rate_ctrl{"wq:rate_ctrl", 9500, 0}; // PX4 inner loop highest priority
```

Comment:
Maybe we should create a new WQ beside this with an enormous stack and leave the original untouched? Either that or restructure to make it possible to set this at runtime.

Reply:
Good point. Is it fine to use the same priority in the new work queue? Is the memory reserved only if a task is added to a work queue?

Reply:
created a new WQ